### DISCUSSION TITLE
Android 15

### DISCUSSION THREAD

**Discussion by rwhelan** [agronick/aa-torque#162] [General]
Has anyone upgraded their pixel to Android 15 yet? Any issues or challenges?

After getting it to work on Android 14, I dont have the courage to be the first to upgrade....

**Comment by Tomanic-glitch** [comment]
I don't have Pixel phone but it works flawlessly on my Android 15 Vivo X100 Pro

**Comment by Coops100** [comment]
> Has anyone upgraded their pixel to Android 15 yet? Any issues or challenges?
> 
> After getting it to work on Android 14, I dont have the courage to be the first to upgrade...

I updated my 6P a few weeks ago to 15 with no problems.

**Comment by lockem567** [comment]
New Pixel8a with Android 15 as the initial OS here with new install of aatorque and nothing appears to work.  King Installer failed to install the aatorque.apk as Playstore or unknown.  Without that, I do not think AA will run the app.

**Comment by impala454** [comment]
I just got a Pixel 9 Pro (with Android 15 pre-installed) and installed aa-torque just fine as per the directions (following the youtube video in the readme).  I just can't get it to show up on AA at all.  The readme says 
```You can find the AA Torque in the menu on the lower right in Android Auto, the one with the dashboard clock on it.```

my clock on the lower right has my battery percentage and such but when I tap it it just shows whether I have notifications or not.  AA torque also doesn't show up in the apps list anywhere.  Not sure what I'm doing wrong.

**Comment by lockem567** [comment]
I just gave up on AA torque and got DashOBD.  DashOBD is a Google approved AA app available in the playstore, so no installation hassles.  It works, although it is not very user friendly..

So why isn’t AA torque also a valid Google AA app?

 

AA torque does not show up as installed by Playstore even when installed by the KingInstaller app.  I do not see any AA torque app button to select and the clock icon does not give me AA torque either.  I think without it listed as installed by playstore, AA torque is not going to show up.

 

--

Mike Locke California Gold

 <http://www.calgoldcoin.com> www.calgoldcoin.com

***@***.***

**Reply by palmtreefrb** [reply]
Looking at the DashOBD on the Playstore. The description does not indicate what the displayed output looks like. Can you customize the display similar to AA Torque?

AA Torque
![20240131_100151](https://github.com/user-attachments/assets/9b228f9d-b3a1-4f3e-b254-be62569b698b)

**Comment by lockem567** [comment]
DashOBD can only display text in AA.  So measurement name on one line with the measured value beneath that and some white space before the next measurement.

DashOBD has a bad habit of “forgetting” what it was last doing when the AA context changes.  So for example if I switch the DNA mode on my Alfa while DashOBD is showing readings and Alfa momentarily takes over the screen in order to show the mode change, when the screen is restored to AA,  DashOBD revert to its home screen instead of the gauge display (DashOBD home screen allows selection between gauges or DTC).  As I said, it is not ideal but it is 0 hassle to install.

 

I am willing to pay for an app that gives a proper gauge display on AA, but I don’t see any such thing.  DashOBD does not advertise that it is an AA app; I found that out via a websearch.  I think there is some missing market opportunity here.

 

--

Mike Locke California Gold

 <http://www.calgoldcoin.com> www.calgoldcoin.com

***@***.***

**Reply by agronick** [reply]
This is a hack and Google would never approve it for the play store.

**Comment by lockem567** [comment]
But I think only the installation is a hack and if the code was submitted to google that hack could be removed.

Compare with DashOBD which is approved by Google and is a pretty sloppy, incomplete implementation.

 

--

Mike Locke California Gold

 <http://www.calgoldcoin.com> www.calgoldcoin.com

***@***.***

**Comment by stefan-sherwood** [comment]
I see two people who report success in Android 15 and lots of people who don't see the Torque app in the list, even after correctly following the instructions. Are we all blind or is there some trick for getting it working?

**Comment by TylerHicks21** [comment]
Has anyone got this working on Android 15?
I am having the issue listed by many, correct install, multiple time. AaTorque not showing in AA app list. 
Anyone have a work around or something to try?
Pixel 8, Android 15.
